stdlib/crypto/src/main/ballerina/src/crypto/crypto.bal

```diff
@@ -218,121 +229,132 @@ function externCrc32b(byte[] input) returns handle = @java:Method {
 } external;
 
 # Returns RSA-MD5 based signature value for the given data.
+# ```ballerina byte[]|crypto:Error signature = crypto:signRsaMd5(data, privateKey); ```
 #
 # + input - The content to be signed
 # + privateKey - Private key used for signing
-# + return - The generated signature or `Error` if private key is invalid
+# + return - The generated signature or `crypto:Error` if private key is invalid
 public function signRsaMd5(byte[] input, PrivateKey privateKey) returns byte[]|Error = @java:Method {
     name: "signRsaMd5",
     class: "org.ballerinalang.stdlib.crypto.nativeimpl.Sign"
 } external;
 
 # Returns RSA-SHA1 based signature value for the given data.
+# ```ballerina byte[]|crypto:Error signature = crypto:signRsaSha1(data, privateKey); ```
 #
 # + input - The content to be signed
 # + privateKey - Private key used for signing
-# + return - The generated signature or `Error` if private key is invalid
+# + return - The generated signature or `crypto:Error` if private key is invalid
```

Comment:
```suggestion
# + return - The generated signature or `crypto:Error` if private key is invalid
```
Please refer the template for the correct format.

Reply:
Corrected.